src/features/Calendar/mockData.ts

```diff
@@ -18,6 +18,7 @@ export const mockEvents: CalendarEventType[] = [
     color: '#F4511E',
     textColor: '#FFFFFF',
     opacity: 0.8,
+    date: '19.03.2023',
```

Comment:
Don't use static dates. We would have to update them over time. Create a function or use some from date-fns to keep dates up to date